doc/reference/reference_lua/decimal.rst

```diff
@@ -70,6 +73,14 @@ post-decimal digits is necessary to get 38-digit precision.
     `Lua math library <https://www.lua.org/pil/18.html>`_,
     which returns 2.718281828459.
 
+.. _decimal-is_decimal:
+
+.. function:: is_decimal(string-or-number-or-decimal-number)
+
+    Returns "true" if specified value is decimal and "false" otherwise.
+    For example if a is 123 then ``decimal.is_decimal(a)`` returns "false". 
```

Comment:
```suggestion
    For example if ``a`` is 123 then ``decimal.is_decimal(a)`` returns "false". 
```

Reply:
according to the other functions, we don't use code formatting for a on this page

Reply:
It's hard to quickly skim a in a sentence without proper highlight, you see. :) Please improve all entries on this page.